Index: src/test/java/it/polimi/ingsw/model/commonGoal/CommonGoal6Test.java
===================================================================
diff --git a/src/test/java/it/polimi/ingsw/model/commonGoal/CommonGoal6Test.java b/src/test/java/it/polimi/ingsw/model/commonGoal/CommonGoal6Test.java
deleted file mode 100644
--- a/src/test/java/it/polimi/ingsw/model/commonGoal/CommonGoal6Test.java	
+++ /dev/null	
@@ -1,7 +0,0 @@
-package it.polimi.ingsw.model.commonGoal;
-
-import junit.framework.TestCase;
-
-public class CommonGoal6Test extends TestCase {
-
-}
\ No newline at end of file
