Index: src/main/java/it/polimi/ingsw/interfaces/ViewActions.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package it.polimi.ingsw.interfaces;\r\n\r\nimport it.polimi.ingsw.server.model.Board;\r\n\r\n/**\r\n * Defines all the actions doable on the view.\r\n *\r\n * @author Francesco Ostidich\r\n */\r\npublic interface ViewActions {\r\n\r\n    /**\r\n     * Notifies the view to show the board.\r\n     *\r\n     * @param board is the board to be shown\r\n     * @author Francesco Ostidich\r\n     */\r\n    void displayBoard(Board board);\r\n\r\n}\r\n
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/interfaces/ViewActions.java b/src/main/java/it/polimi/ingsw/interfaces/ViewActions.java
--- a/src/main/java/it/polimi/ingsw/interfaces/ViewActions.java	
+++ b/src/main/java/it/polimi/ingsw/interfaces/ViewActions.java	
@@ -17,4 +17,5 @@
      */
     void displayBoard(Board board);
 
+
 }
